scout/server/blueprints/variants/forms.py

```diff
@@ -164,3 +173,28 @@ class SvFiltersForm(VariantFiltersForm):
     clingen_ngi = IntegerField("ClinGen NGI obs")
     swegen = IntegerField("SweGen obs")
     clinical_filter = SubmitField(label="Clinical filter")
+
+    # polymorphic constant base for clinical filter
+    clinical_filter_base = CLINICAL_FILTER_BASE_SV
+
+
+class CancerSvFiltersForm(SvFiltersForm):
+    """Extends SvFiltersForm for cancer structural variants"""
+
+    depth = IntegerField("Depth >", validators=[validators.Optional()])
+    alt_count = IntegerField("Min alt count", validators=[validators.Optional()])
+    control_frequency = BetterDecimalField(
+        "Normal alt AF <", places=2, validators=[validators.Optional()]
+    )
+    tumor_frequency = BetterDecimalField(
+        "Tumor alt AF >", places=2, validators=[validators.Optional()]
+    )
+
+
+FILTERSFORMCLASS = {
```

Comment:
It should perhaps be a clever dynamic constructor instead, but at least it is better than before.